src/main/java/de/jtheb/fh/lagerverwaltung/entities/Compartment.java

```diff
@@ -42,19 +42,26 @@ public int getItemsVolume() {
         return volume;
     }
 
-    public boolean itemFits(Item item) {
-        this.getArticleNr();
+    public boolean itemFitsWithArticleNr(Item item) {
+        return (this.articleNrFits(item) && itemFitsVolume(item));
```

Comment:
not sure what withArticleNr implies, maybe add javadoc. see example at https://de.wikipedia.org/wiki/Javadoc